docker-compose.yaml

```
@@ -67,7 +67,7 @@ services:
         ports:
             - "5432:5432"
         healthcheck:
-            test: ["CMD-SHELL", "pg_isready -U extract"]
+            test: ["CMD-SHELL", "pg_isready -U extractuser"]
```

You also have to speccify the DB if it is not the same as the used:
```suggestion
            test: ["CMD-SHELL", "pg_isready -U extractuser" -d postgres]
```